scripts/globals/mobskills/reactive_armor.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+-- Reactive Armor
+-- Description: Covers the user in shock spikes. Enemies that hit it take lightning damage.
+-- Type: Enhancing
+-- Utsusemi/Blink absorb: N/A
+-- Range: Self
+-- Notes:
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
```

Comment:
Settings does not appear to be used here, and should not be required if it isn't needed